include/openthread/platform/ble.h

```diff
@@ -385,6 +415,18 @@ typedef struct otBleRadioPacket
     int8_t   mPower;  ///< Transmit/receive power in dBm.
 } otBleRadioPacket;
 
+/**
+ * The enum indicates the outcome of the L2CAP connection request procedure.
+ * @see Bluetooth v5.0 | Vol 3, Part A, 4.23, Table 4.20.
```

Comment:
This doesn't appear to be proper use of the `@see` command, which is used with automatic link generation.

Reply:
Would changing this from "@see" to "See " work?

Reply:
Yes.